engine/standalone/util/src/main/scala/pl/touk/nussknacker/engine/standalone/utils/typed/TypingResultToJsonSchemaConverter.scala

```diff
@@ -0,0 +1,39 @@
+package pl.touk.nussknacker.engine.standalone.utils.typed
+
+import com.typesafe.scalalogging.LazyLogging
+import io.circe.Json
+import io.circe.syntax._
+import pl.touk.nussknacker.engine.api.typed.typing.{TypedClass, TypedObjectTypingResult, TypingResult}
+import pl.touk.nussknacker.engine.util.json.BestEffortJsonEncoder
+
+class TypingResultToJsonSchemaConverter(exampleValues: Map[String, Json]) extends LazyLogging {
```

Comment:
1) Where this class is used? 2) I don't see usage of logger

Reply:
As I wrote to mproch I changed my conception a little bit and decided to remove this class. As you observed correctly it had no use in this PR.